### DISCUSSION TITLE
Livewire updatedFoo not triggering on tests with PestPHP

### DISCUSSION THREAD

**Discussion by spooksie** [livewire/livewire#4066] [1. Help]
Hi,

Im writing Pest tests for Livewire components, which works great. However I'm hitting a brick wall with a new component that uses the fooUpdated() functions.

Pest Test
```
it('change sections', function () {
    $component = livewire(Products::class)
    ->set('section', "categories")
```

Livewire 
```
public function updatedSection()
    {
        $this->clearCategories();

        if ($this->section == 'categories') {
            $this->categories['sites'] = services_sites::get()->keyBy('id')->toArray();
            $this->loadGroupsData();
            $this->loadServicesData();
            $this->loadPanelssData();
        }
    }
```
When I change the section It doesn't trigger the function at all. On live it works as intended.

**Comment by joshhanley** [comment]
@spooksie any chance you could test that with a standard phpunit test as well?

It generally works for me, but I don't use pest, so would be good to narrow down whether there might be an issue with the pest Livewire helpers or something else.

Might also be worth putting a `dump()` in the updatedSection method and seeing if that fires at all.

Hope this helps!
